src/btree/bt_vrfy.c

```diff
@@ -585,7 +589,16 @@ __verify_tree(
 
             /* Verify the subtree. */
             ++vs->depth;
-            WT_RET(__wt_page_in(session, child_ref, 0));
+            ret = __wt_page_in(session, child_ref, 0);
+
+            /*
+             * If configured, continue traversing through the pages of the tree even after
+             * encountering errors in the verification process.
+             */
+            if (vs->read_corrupt)
+                continue;
```

Comment:
I think this looks good generally. One concern I have is retaining the original error. It seems like it may get forgotten (although I have not looked at the surrounding code and the rest of the loop this is in). We want to make sure we return the same error information as when it stops on the first error. (Same comment for other location too.)

Reply:
I see what you mean, originally I wasn't sure if we wanted to retain this error or not. I'm thinking we could maybe store this error as a field in the `vs` structure. Then maybe the caller of `__verify_tree` can check for this error along with any other errors `__verify_tree` has returned? I can think over this more today.